src/app/gamification/shared/tour.interceptor.ts

```diff
@@ -73,7 +73,7 @@ export class TourInterceptor implements HttpInterceptor, OnInit {
   private shouldActiveTour(): boolean {
     let activateTour: boolean;
     if (_.isNil(this.profile)) {
```

Comment:
@leandronunes eu percebi que em alguns lugares existe o `this.profile` mas é um objeto vazio... nestes lugares eu mudei o condicional para `this. profile && this.profile.id`.